tests/components/switcher_kis/test_init.py

```diff
@@ -0,0 +1,47 @@
+"""Test cases for the switcher_kis component."""
+
+from typing import Any, Generator
+
+from homeassistant.components.switcher_kis import (DOMAIN, DATA_DEVICE)
+from homeassistant.helpers.typing import HomeAssistantType
+from homeassistant.setup import async_setup_component
+
+from .consts import (
+    DUMMY_AUTO_OFF_SET, DUMMY_DEVICE_ID, DUMMY_DEVICE_NAME,
+    DUMMY_DEVICE_STATE, DUMMY_ELECTRIC_CURRENT, DUMMY_IP_ADDRESS,
+    DUMMY_MAC_ADDRESS, DUMMY_PHONE_ID, DUMMY_POWER_CONSUMPTION,
+    DUMMY_REMAINING_TIME, MANDATORY_CONFIGURATION)
+
+
+async def test_failed_config(hass: HomeAssistantType) -> None:
+    """Test failed configuration."""
+    assert await async_setup_component(
+        hass, DOMAIN, MANDATORY_CONFIGURATION) is False
+
+
+async def test_minimal_config(hass: HomeAssistantType,
+                              mock_bridge: Generator[None, Any, None]
+                              ) -> None:
+    """Test setup with configuration minimal entries."""
+    assert await async_setup_component(hass, DOMAIN, MANDATORY_CONFIGURATION)
+
+
+async def test_discovery_data_bucket(
+        hass: HomeAssistantType,
+        mock_bridge: Generator[None, Any, None]
+        ) -> None:
+    """Test the event send with the updated device."""
+    assert await async_setup_component(
+        hass, DOMAIN, MANDATORY_CONFIGURATION)
+
```

Comment:
```python
await hass.async_block_till_done()
```
Ensure platform loaded

Reply:
Fixed in this [commit](https://github.com/home-assistant/home-assistant/pull/22325/commits/0557f5a60d8da3a9e124771a3aafcf218a15c5a3).